Civi/CiviAuth0Jwt/CheckAuth0JwtCredential.php

```diff
@@ -80,10 +80,8 @@ public function bearerAuth0Jwt(CheckCredentialEvent $checkEvent): void {
       }
       catch (CryptoException $e) {
         // Not a valid AuthX JWT. Proceed to check any other token sources.
-        Logger::debug("IGNORED, decode failed (CryptoException): " . $e->getMessage());
       }
       catch (\Exception $e) {
-        Logger::debug("IGNORED, decode failed (Exception): " . $e->getMessage());
```

Comment:
Might want to add a comment in the code at least explaining that errors of this type are specifically ignored, and the reason for that.